src/main/java/bio/terra/cli/context/utils/Printer.java

```diff
@@ -0,0 +1,89 @@
+package bio.terra.cli.context.utils;
+
+import java.io.PrintStream;
+import java.io.PrintWriter;
+import java.nio.charset.Charset;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+
+/**
+ * Singleton class for holding a reference to output streams (e.g. stdout, stderr). The purpose of
+ * holding these references in a single place is so that we can write output throughout the codebase
+ * without passing around the output streams from the top-level command classes.
+ */
+public class Printer {
```

Comment:
Are you not using spring in the CLI?

Reply:
No, the CLI doesn't use Spring. Why do you ask?